client/cypress/e2e/_16allocationAWOpen.cy.ts

```diff
@@ -0,0 +1,332 @@
+            .should('eq', 'Confirm');
+          cy.get('[data-test=AllocationNavigation__ctaButton]').click();
+          cy.get('[data-test=ModalAllocationLowUqScore]').should('be.visible');
+          cy.get('[data-test=AllocationLowUqScore__checkbox]').check();
+          cy.get('[data-test=AllocationLowUqScore__ctaButton]').click();
+          cy.get('[data-test=ModalAllocationLowUqScore]').should('not.exist');
+          cy.wait(500);
+          cy.get('[data-test=AllocationNavigation__ctaButton]').should('be.disabled');
+          cy.get('[data-test=AllocationNavigation__ctaButton]')
+            .invoke('text')
+            .should('eq', 'Waiting');
+          cy.confirmMetamaskDataSignatureRequest();
+          cy.wait(1000);
+          cy.get('[data-test=AllocationSummary]').should('exist');
+          cy.get('[data-test=AllocationSummaryProject]').should('have.length', 3);
+          cy.wait(1000);
+          cy.get('[data-test=AllocationSummary__totalDonated]').scrollIntoView();
+          cy.get('[data-test=AllocationSummary__totalDonated]').should('be.visible');
+          cy.get('[data-test=AllocationSummary__totalDonated__value--loading]').should('not.exist');
+          cy.get('[data-test=AllocationSummary__totalDonated__value]').then($totalDonated => {
+            expect(parseFloat(`${$totalDonated.text()}`.replace('<', ''))).to.be.gt(0);
+          });
+
+          cy.get('[data-test=AllocationSummary__matchFunding]').scrollIntoView();
+          cy.get('[data-test=AllocationSummary__matchFunding]').should('be.visible');
+          cy.get('[data-test=AllocationSummary__matchFunding__value--loading]').should('not.exist');
+          cy.get('[data-test=AllocationSummary__matchFunding__value]').then($matchFunding => {
+            expect(parseFloat(`${$matchFunding.text()}`.replace('<', ''))).to.be.gt(0);
+          });
+          cy.get('[data-test=AllocationSummary__totalImpact]').scrollIntoView();
+          cy.get('[data-test=AllocationSummary__totalImpact]').should('be.visible');
+          cy.get('[data-test=AllocationSummary__totalImpact__value]').then($totalImpact => {
+            expect(parseFloat(`${$totalImpact.text()}`.replace('<', ''))).to.be.gt(0);
+          });
+
+          cy.get('[data-test=AllocationSummary__personalRewardBox]').scrollIntoView();
+          cy.get('[data-test=AllocationSummary__personalRewardBox]').should('be.visible');
+          cy.get('[data-test=AllocationSummary__personalReward--loading]').should('not.exist');
+          cy.get('[data-test=AllocationSummary__personalReward]').then($personalReward => {
+            expect(parseFloat(`${$personalReward.text()}`.replace('<', ''))).to.be.gt(0);
+          });
+
+          cy.get('[data-test=AllocationSliderBox__Slider__thumb]').scrollIntoView();
+          cy.get('[data-test=AllocationSliderBox__Slider__thumb]').should('not.be.visible');
+          cy.get('[data-test=AllocationNavigation__ctaButton]').invoke('text').should('eq', 'Edit');
```

Comment:
Suggestion, but it would require change in the component too.
```suggestion
          cy.get('[data-test=AllocationNavigation__Button--cta]').invoke('text').should('eq', 'Edit');
```

Reply:
Done